### DISCUSSION TITLE
增加后台运行保留弹幕功能

### DISCUSSION THREAD

**Discussion by ghost** [the1812/Bilibili-Evolved#4194] [功能建议 / Ideas]
经常把B站放到后台听视频，听到精彩的地方想回来看弹幕，但是B站在后台一段时间后会关闭弹幕功能，返回页面之后弹幕是空的看不到当前弹幕，希望能加一个禁用此特性的开关

**Comment by the1812** [comment]
不会空啊, 你配置太低了?
